charts/rasa-x/templates/rasa-x-deployment.yaml

```diff
@@ -34,6 +34,11 @@ spec:
       tolerations:
 {{ toYaml .Values.rasax.tolerations | indent 6 }}
       {{- end }}
+
+      {{- if .Values.rasax.initContainers }}
+      initContainers:
```

Comment:
The same problem is here, the `initContainers` is duplicated

Reply:
agh I swear I got rid of it my bad